src/lib/Map.svelte

```diff
@@ -56,52 +57,69 @@
 					'icon-allow-overlap': true,
 					'icon-image': 'marker',
 					'icon-size': 0.5,
-					"icon-anchor": "bottom"
+					'icon-anchor': 'bottom'
 				},
 				paint: {}
 			});
 
 			map.on('click', 'moments-layer', function (e) {
-        if (!e.features) {
-          return;
-        }
-        if (e.features.length === 0) {
-          return;
-        }
-
-        const feature = e.features[0];
-
-        if (feature.geometry.type !== 'Point') {
-          return;
-        }
-        
-        const coordinates = (feature.geometry as GeoJSON.Point).coordinates;
-        getMoment(feature.properties.id)
-          .then((text) => {
-            const description = text;
-            if (coordinates.length === 2) {
-              new Popup({offset: 40})
-                .setLngLat(coordinates as LngLatLike)
-                .setHTML(description)
-                .addTo(map);
-            } else {
-              console.error('Invalid coordinates format');
-            }
-          })
-          .catch((error) => {
-            console.error('Error fetching moment:', error);
-          });
+				isMomentLayerClicked = true;
+				if (e.features && e.features.length > 0) {
```

Comment:
Niels refactored this part in a small PR (#28) to be less indented and merged it into main. Do you have a preference between the styles @narcode?

Reply:
I did a small refactor based on Niels' approach to the map click event.